sw/device/lib/testing/meson.build

```diff
@@ -0,0 +1,78 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+src_root_dir = meson.source_root()
```

Comment:
Meson variables (which look file local) are actually global (I know, right?).

I would avoid variables if possible, and, when doing so would make the file unreadable, consider a prefix like `sw_testing_`.

Reply:
Sure, I inlined two variable and added a prefix to the remaining ones.